src/client/utils.ts

```diff
@@ -0,0 +1,8 @@
+import { strToBytes } from '../basicElements'
+
+export function formatDatastoreKey(key: Uint8Array | string): Uint8Array {
+  if (typeof key === 'string') {
+    return strToBytes(key)
+  }
+  return key
+}
```

Comment:
can you move this in `src/utils/formatter.ts`
It is the same idea of `parseCallArgs` and it could be used in different contexts